apache-Dockerfile-block-1

```diff
@@ -5,7 +5,7 @@ RUN rm -rf /var/www/html && mkdir -p /var/lock/apache2 /var/run/apache2 /var/log
 # Apache + PHP requires preforking Apache for best results
 RUN a2dismod mpm_event && a2enmod mpm_prefork
 
-RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist
+RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist && rm /etc/apache2/{conf,sites}-enabled/*
```

Comment:
Dough. Was sure I did test that (but compiling the Dockerfile manually). I removed the bashism in a follow-up commit to the same branch, it already is included in the pull commit. If you want I can also merge the commits and force-push, but it would probably break the whole discussion here.